bin/pygrb/pycbc_pygrb_page_tables

```diff
@@ -44,173 +45,123 @@ __version__ = pycbc.version.git_verbose_msg
 __date__ = pycbc.version.date
 __program__ = "pycbc_pygrb_page_tables"
 
-
 # =============================================================================
 # Functions
 # =============================================================================
-# Function to load trigger data
-def load_data(input_file, vetoes, ifos, opts, injections=False):
-    """Load data from a trigger/injection file"""
-
-    # Initialize the dictionary
-    data = {}
-    data['trig_time'] = None
-    data['coherent'] = None
-    data['reweighted_snr'] = None
-
-    if input_file:
-        if injections:
-            logging.info("Loading injections...")
-            # This will eventually become ppu.load_injections()
-            trigs_or_injs = ppu.load_triggers(input_file, vetoes)
-        else:
-            logging.info("Loading triggers...")
-            trigs_or_injs = ppu.load_triggers(input_file, vetoes)
-            
-        data['trig_time'] = trigs_or_injs['network/end_time_gc'][:]
-        num_trigs_or_injs = len(data['trig_time'])
-        data['coherent'] = trigs_or_injs['network/coherent_snr'][:]
-        data['reweighted_snr'] = trigs_or_injs['network/reweighted_snr'][:]
-
-        if injections:
-            logging.info("%d injections found.", num_trigs_or_injs)
-        else:
-            logging.info("%d triggers found.", num_trigs_or_injs)
```

Comment:
Please make sure you have this information for the user being printed somewhere.